scripts/scripts.js

```diff
@@ -154,7 +160,7 @@ function isArticlePage() {
 // eslint-disable-next-line no-unused-vars
 function buildAutoBlocks(main) {
   try {
-    // buildHeroBlock(main);
+    buildBlogHeader(main);
```

Comment:
This is the right idea for autoblocking, but the `buildArticleHeader` below also prepends to `main`, so the result looks incorrect. You'll have to adjust the article header to inject its section after your blog header, see for example:

https://9-blog-header--servicenow--hlxsites.hlx.live/blogs/2023/demo/maximize-efficiency-hyperautomation